### ISSUE TITLE
Map pins: tooltip and popup

### ISSUE THREAD

**Issue by ekes** [localgovdrupal/localgov_directories#337]
What's the preferred output here?

Tooltip: This is when you hover over a pin. Should there be one? What should be in it? Title? Clickable?
Popup: This is when you click on a pin. Should there be one? What should be in it? More than just the title?

Presently we just have a Tooltip with a title and link as default. Came up in discussion with George (North Devon) wanting to change it. We've already changed it at Greenwich to: Tooltip (Title without link) Popup (Rendered View Fields: which are Title as link, and body).

Taking into account desktop & mobile in one configuration.

**Comment by GC-92** [comment]
I would propose for the Popup feature to be default.

Hover to view the title of the directory is limited to mouse and click users and doesn't offer anything for mobile users (unless you press and hold over the icon, which doesnt offer much than just the title?)

Based on our user research which is majority mobile users of our website and our AOB feedback, we implemented the popup map pin information after their request and they approved of the decision after some testing.

I also believe this would appeal to other councils as it allows the map to be more interactive rather than being limited for mouse/tap and hold hover.

Also the process to enable the popup the instructions may seem complicated to follow for other website managers to complete.

My only other suggestion is how can I make the popup box width size fit better for mobile devices?
I noticed this once testing with with mobile dimensions. This may be due to my teaser/summary text settings I am using for my directory listings.

See my screen shot:
![image](https://github.com/localgovdrupal/localgov_directories/assets/110611127/37f10f94-d5c4-4219-a235-209930fd959c)

Kind regards,
